hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1179,6 +1179,10 @@ func (r *NodePoolReconciler) listMachineTemplates(nodePool *hyperv1.NodePool) ([
 		if err != nil {
 			return nil, err
 		}
+	default:
+		//need a default path that returns a value that does not cause the hypershift operator to crash
```

Comment:
nit: can we please leave space after `//` https://go.dev/doc/effective_go#commentary